samples/wifi/shell/README.rst

```diff
@@ -115,11 +115,15 @@ Supported CLI commands
        | [-h, --help] : Print out the help for the scan command.
    * - connect
      - | Connect to a Wi-Fi AP with the following parameters:
-       | <SSID>
-       | <Channel number> (optional: 0 means all)
-       | <PSK> (optional: valid only for secured SSIDs)
-       | <Security type> (optional: 0-None, 1-PSK, 2-PSK-256, 3-SAE)
-       | <MFP> (optional: 0-Disable, 1-Optional, 2-Required)
+       | "<SSID>"
+       | [channel number/band: > 0:Channel, 0:any channel,
```

Comment:
Is it required to mention it twice 0:Channel, 0:Any channel?
```suggestion
       | [Channel number/band: > 0:Channel, 0:Any channel,
```

Reply:
It's `> 0`  and `0`, both are different.